src/constants.ts

```diff
@@ -1,17 +1,31 @@
-import { assertNotNull } from "@subsquid/evm-processor";
+import { assertNotNull } from '@subsquid/evm-processor';
+
+const SQUID_NETWORK = process.env.SQUID_NETWORK || 'eth-sepolia';
+
+const IS_PRODUCTION = SQUID_NETWORK === 'optimism-mainnet';
```

Comment:
I am uneasy with limiting our env configuration to two production and non-production types. I think a more flexible one would be naming each env like (op-sepolia-staging, op-mainnet-production, sepolia-mainnet-staging, ...) and in the config we change only one variable. Then we can have multiple configurations with more readability.

Reply:
Let's stick to this for fast follow